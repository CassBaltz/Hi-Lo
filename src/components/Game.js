import React from 'react';

import Player from './Player';
import Table from './Table';
import { PLAYER_ONE, PLAYER_TWO, } from '../redux/constants';

export const Game = () => {
  return (
    <div>
      <Player playerKey={PLAYER_ONE} />
      <Table />
      <Player playerKey={PLAYER_TWO} rotateAvatar={true} />
    </div>
  );
};

export default Game;