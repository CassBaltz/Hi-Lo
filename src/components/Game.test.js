import React from "react";
import Game from "./Game";

describe("<Game />", () => {
  it("should render two players and a table component", () => {
    const game = shallow(<Game />);
    expect(toJson(game)).toMatchSnapshot();
  });
});