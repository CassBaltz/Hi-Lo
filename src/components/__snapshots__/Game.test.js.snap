// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> should render two players and a table component 1`] = `
<div>
  <Connect(WithStyles(Player))
    playerKey="PLAYER_ONE"
  />
  <Connect(WithStyles(Table)) />
  <Connect(WithStyles(Player))
    playerKey="PLAYER_TWO"
    rotateAvatar={true}
  />
</div>
`;
